src/extensions/runtimes/node.ts

```diff
@@ -482,7 +483,7 @@ export async function writeSDK(
   // Build it
   logLabeledBullet("extensions", `running 'npm --prefix ${shortDirPath} run build'`);
   try {
-    execFileSync("npm", ["--prefix", dirPath, "run", "build"]);
+    await spawnWithOutput("npm", ["--prefix", dirPath, "run", "build"]);
```

Comment:
Nit: Since we don't use the output, should we be using wrapSpawn in this file instead?

Reply:
wrapSpawn requires project directory which is not available here. We may not use the output, but it doesn't really matter. Maybe in future I can refactor wrapSpawn to have project directory be optional.